pkg/aggregator/mocksender/mock_test.go

```diff
@@ -0,0 +1,51 @@
+package mocksender
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/metrics"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExpectedInActual(t *testing.T) {
+	assert.True(t, expectedInActual([]string{}, []string{}))
+	assert.True(t, expectedInActual([]string{}, []string{"one"}))
+	assert.True(t, expectedInActual([]string{"one"}, []string{"one"}))
+	assert.True(t, expectedInActual([]string{}, []string{"one", "two"}))
+	assert.True(t, expectedInActual([]string{"one"}, []string{"one", "two"}))
+	assert.True(t, expectedInActual([]string{"one", "two"}, []string{"one", "two"}))
+	assert.True(t, expectedInActual([]string{"one", "two"}[0:0], []string{"one", "two"}))
```

Comment:
the  `[ ]` construct is good if your array is already allocated somewhere. either directly allocate what you want or pre-allocate []string{"one", "two", "three"} in a var an reuse it